ORStools/proc/directions_points_layers_proc.py

```diff
@@ -211,7 +214,14 @@ def sort_end(f):
         return {self.OUT: dest_id}
 
     @staticmethod
-    def _get_route_dict(source, source_field, sort_start, destination, destination_field, sort_end):
+    def _get_route_dict(
+        source: QgsProcessingParameterFeatureSource,
+        source_field: QgsField,
+        sort_start,
+        destination: QgsProcessingParameterFeatureSource,
```

Comment:
```suggestion
        destination: QgsFeatureSink,
```

Again, this might be wrong 😁

Reply:
Here as well, Printing the type gives me `QgsProcessingFeatureSource`.